docs/index.md

```diff
@@ -102,13 +102,13 @@ package.json usage
 
 | Name        | Type                                | Default                    | Description                                                                    |
 | ----------- | ----------------------------------- | -------------------------- | ------------------------------------------------------------------------------ |
-| srcDir      | `string`                            |                            | Directory to hold NW app files                                                 |
-| mode        | `run \| build`                      |                            | Run or build application                                                       |
-| version     | `latest \| stable \| string \| lts` |                            | NW runtime version                                                             |
-| flavour     | `sdk \| normal`                     |                            | NW runtime build flavour.                                                      |
-| platform    | `linux \| osx \| win`               |                            | NW supported platforms                                                         |
-| arch        | `ia32 \| x64`                       |                            | NW supported architectures                                                     |
-| outDir      | `string`                            |                            | Directory to store build artifacts                                             |
+| srcDir      | `string`                            | `./`                       | Directory to hold NW app files                                                 |
+| mode        | `run \| build`                      | `build`                    | Run or build application                                                       |
+| version     | `latest \| stable \| string \| lts` | `latest`                   | NW runtime version                                                             |
+| flavour     | `sdk \| normal`                     | `sdk`                      | NW runtime build flavour.                                                      |
```

Comment:
Since the default changed in the code, it should be reflected in the documentation
```suggestion
| flavour     | `sdk \| normal`                     | `normal`                   | NW runtime build flavour.                                                      |
```